src/pages/Compare/index.spec.tsx

```diff
@@ -112,15 +113,25 @@ describe(__filename, () => {
   };
 
   const _loadDiff = ({
-    addonId = 1,
-    baseVersionId = 2,
-    headVersionId = 3,
+    addonId = nextUniqueId(),
+    baseVersionId = nextUniqueId(),
+    headVersionId = baseVersionId + 1,
```

Comment:
I wanted to make it clear that `headVersionId` must be larger than `baseVersionId` for this test to work. Otherwise, someone could rearrange the definitions and become confused as to why it breaks.